scripts/index.js

```diff
@@ -0,0 +1 @@
+export {initNavDropdowns} from '../templates/docs/examples/patterns/navigation/_script-sliding.js';
```

Comment:
we have `index.scss` in root folder, I guess it's fine to have `index.js` there as well for exporting.